python/marvin/tools/map.py

```diff
@@ -386,3 +417,257 @@ def plot(self, array='value', xlim=None, ylim=None, zlim=None,
             return (ax, fig)
         else:
             return ax
+
+    def _make_image(self, snr_min, log_cb, use_mask):
+        """Make masked array of image.
+
+        Args:
```

Comment:
I like Parameters because it contains both things that are args and kwargs.  If we title something as Args: that makes me only think of parameters that are args.  And I'll want to do an Args: and Kwargs:.  I suggest we commit to what we already have. :)